qiskit/providers/backend.py

```diff
@@ -229,3 +241,443 @@ class can handle either situation.
             Job: The job object for the run
         """
         pass
+
+
+class QubitProperties:
+    """A representation of the properties of a qubit on a backend.
+
+    This class provides the minimum expected optional properties that a backend
```

Comment:
I can tweak the language to be more explicit I thought `minimal expected optional properties` to cover this kind of thing, heh but probably just in my head. This is really saying that these are the fields that things elsewhere in terra can use for optional features. They're not required to be present (and they default to `None` because of this) but it's the optional stuff terra can use today. This is to distinguish it from custom things for a particular backend or qubit type which can be added via a subclass (or jammed in the `properties` dict). Do you have any suggestions?

Reply:
I tweaked the wording a bit in: https://github.com/Qiskit/qiskit-terra/pull/5885/commits/2942118b0d441757aaeef1333f72cbdf53e05452 does this look better?

Reply:
Yes, just one minor typo `howeverm` -> `however`.

Reply:
Ignore, its been corrected in another qubit.